.changeset/forty-spies-heal.md

```diff
@@ -0,0 +1,6 @@
+---
+"@guardian/cdk": minor
+---
+
+- Load balancers now add headers with information about the TLS version and cipher suite used during negotiation
+- Load balancers now drop invalid headers before forwarding requests to the target
```

Comment:
> Load balancers now drop invalid headers before forwarding requests to the target

This sounds like it might be a breaking change if anybody relies on custom headers! Maybe this should be a major version bump?

Reply:
Custom headers (`X-etc-etc`) are considered valid, so they won't be dropped. 

From the AWS docs about this setting:

> Elastic Load Balancing requires that valid HTTP header names conform to the regular expression [-A-Za-z0-9]+, as described in the HTTP Field Name Registry. Each name consists of alphanumeric characters or hyphens. Select true if you want HTTP headers that do not conform to this pattern, to be removed from requests.

Do you still think this should be a major version bump?

Reply:
Ah gotcha. Highly doubtful that anybody would be using such a header, but I'd still suggest this be a major bump just in case!

Reply:
And could we possibly add the description of what is a valid header to the release notes, just so nobody makes the same mistake i did!